assets/js/components/notifications/UserInputSuccessNotification.js

```diff
@@ -20,14 +20,24 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { useEffect, useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import Notification from '../legacy-notifications/notification';
 import UserInputSuccessImage from '../../../svg/congrats.svg';
+import { trackEvent } from '../../util';
 
 export default function UserInputSuccessNotification() {
+	useEffect( () => {
+		trackEvent( 'user_input', 'success_notification_view' );
+	}, [] );
+
+	const handleOnDismiss = useCallback( () => {
```

Comment:
No need for `useCallback` here I think.

cc @aaemnnosttv @tofumatt I don't recall exactly, but what was our guidance for whether or not using `useCallback`? I remember there was a conversation about it at some point.

Reply:
@felixarntz the main reason to use it is when the function itself is compared for reference equality in a component, e.g. in a dependency list or when used for comparison in `shouldComponentUpdate` although that's more specific to 3p components since ours are all functional now. Since `Notification` is our own component, it should be fine to omit here.

Larger conversation is here: https://github.com/google/site-kit-wp/issues/2351